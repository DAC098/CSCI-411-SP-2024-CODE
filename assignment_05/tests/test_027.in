421 356
9
27
29
42
49
86
184
185
216
242
255
255
281
285
297
324
370
373
380
409
415
499
574
575
593
599
615
641
641
728
730
752
752
754
792
860
882
959
961
970
1032
1062
1064
1071
1083
1091
1095
1099
1115
1126
1156
1166
1222
1237
1264
1275
1289
1328
1356
1387
1412
1419
1424
1432
1432
1439
1450
1451
1461
1538
1556
1599
1647
1682
1716
1723
1779
1784
1803
1830
1834
1896
1897
1917
1932
1934
1952
1992
2049
2065
2087
2099
2133
2137
2149
2150
2174
2220
2257
2286
2330
2355
2381
2387
2393
2429
2448
2465
2473
2487
2487
2575
2612
2634
2636
2661
2681
2755
2757
2806
2833
2851
2858
2923
2930
2932
2933
2935
2948
2971
2971
2983
3009
3011
3105
3111
3121
3137
3163
3168
3207
3280
3286
3315
3370
3379
3397
3429
3512
3545
3571
3603
3621
3623
3790
3809
3844
3854
3900
3902
3924
3969
3978
3989
4030
4031
4061
4065
4097
4115
4127
4170
4194
4206
4275
4300
4369
4372
4375
4396
4402
4420
4477
4500
4512
4520
4531
4535
4548
4613
4634
4658
4689
4696
4713
4761
4769
4773
4789
4829
4840
4869
4884
4971
4985
4989
5025
5033
5043
5138
5161
5173
5191
5202
5245
5304
5304
5343
5376
5384
5396
5530
5560
5583
5633
5667
5731
5741
5748
5772
5800
5806
5809
5814
5822
5836
5849
5927
5929
5941
5955
5959
5969
6010
6063
6077
6090
6095
6144
6156
6167
6182
6187
6192
6198
6199
6218
6262
6320
6363
6369
6447
6453
6460
6486
6502
6516
6530
6534
6543
6564
6566
6592
6608
6616
6624
6648
6651
6656
6684
6698
6800
6832
6847
6851
6883
6883
6905
6918
6950
6969
6973
6978
7001
7048
7081
7092
7112
7189
7221
7237
7247
7252
7285
7305
7337
7359
7370
7381
7425
7496
7500
7526
7553
7558
7563
7622
7625
7675
7697
7707
7734
7743
7744
7774
7824
7868
7879
7880
7897
7931
7943
7943
7968
7986
8005
8008
8048
8117
8124
8159
8159
8186
8206
8207
8216
8218
8225
8229
8258
8291
8293
8322
8323
8352
8364
8384
8386
8408
8425
8427
8442
8564
8652
8726
8729
8741
8753
8769
8782
8801
8814
8826
8865
8869
8907
8974
9017
9022
9025
9039
9056
9057
9064
9108
9124
9139
9160
9172
9189
9245
9257
9258
9300
9332
9335
9377
9395
9400
9412
9420
9434
9437
9479
9493
9544
9546
9550
9576
9618
9654
9657
9675
9686
9775
9823
9853
9902
9906
9946
9990
