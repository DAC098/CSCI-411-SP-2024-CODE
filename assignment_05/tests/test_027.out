Enter the number of hotels and the ideal number of miles to travel per day: Enter 421 hotel distances each on a separate line: Hotels to visit: 17 30 45 64 78 94 110 122 141 151 161 176 192 207 220 228 248 264 283 299 313 327 346 363 376 393 410 421 
Total penalty: 2476
